models/issue.go

```diff
@@ -1156,17 +1157,20 @@ func (opts *IssuesOptions) setupSession(sess *xorm.Session) {
 
 	if opts.ProjectID > 0 {
 		sess.Join("INNER", "project_issue", "issue.id = project_issue.issue_id").
-			And("project_issue.project_id=?", opts.ProjectID)
+			And("project_issue.project_id=?", opts.ProjectID).OrderBy("priority")
```

Comment:
Probably sensible to add `project_issue` to that `ORDER BY  priority`
```suggestion
			And("project_issue.project_id=?", opts.ProjectID).OrderBy("`project_issue`.priority")
```